lib/solargraph/shell.rb

```diff
@@ -182,7 +187,45 @@ def list
       puts "#{workspace.filenames.length} files total."
     end
 
-    private
+    desc 'cache', 'Cache a gem', hide: true
+    # @return [void]
+    # @param gem [String]
+    # @param version [String, nil]
+    def cache gem, version = nil
+      spec = Gem::Specification.find_by_name(gem, version)
+      pins = GemPins.build(spec)
+      Cache.save('gems', "#{spec.name}-#{spec.version}.ser", pins)
+    end
```

Comment:
@castwide: Were 'cache' and 'gems' intended to be removed in #790?

Reply:
@apiology Nope, I dropped them by accident while fixing a merge conflict. They're back on master now.